json-rpc/types/src/views.rs

```diff
@@ -363,7 +411,7 @@ impl From<Transaction> for TransactionDataView {
                     public_key: hex::encode(t.authenticator().public_key_bytes()),
                     sequence_number: t.sequence_number(),
                     max_gas_amount: t.max_gas_amount(),
-                    gas_unit_price: t.gas_unit_price(),
+                    gas_unit_price: AmountView::new(t.gas_unit_price(), t.gas_currency_code()),
```

Comment:
let's add an field called gas_currency instead changing the existing field  (which will break a lot of client)